cylc/flow/task_pool.py

```diff
@@ -1119,17 +1119,25 @@ def check_abort_on_task_fails(self):
         """
         return self.abort_task_failed
 
-    def spawn_on_output(self, itask, output):
+    def spawn_on_output(self, itask, output, forced=False):
         """Spawn and update itask's children, remove itask if finished.
 
         Also set a the abort-on-task-failed flag if necessary.
-        If not reflow update existing children but don't spawn them.
+
+        If not reflow:
+            - update existing children but don't spawn new ones
+            - unless forced (manual command): spawn but with no flow number
 
         If an absolute output is completed update the store of completed abs
         outputs, and update the prerequisites of every instance of the child
         in the pool. (And in self.spawn() use the store of completed abs
         outputs to satisfy any tasks with abs prerequisites).
 
+        Args:
+            tasks: List of identifiers, see `task globs`
```

Comment:
Where am I looking to see about task globs?
Is this markdown or rst?
- If RST use `` or a full `:ref:`

Reply:
Haha, good question - I pasted that line from another method with the same arg list.  I'll take a look.